### PR TITLE
Oracle SOA Suite Chart

### PR DESCRIPTION
**Pull request by streamnsight**
Also adding a script to package charts and publish.
Publishing will require updating the Github page to point to the `main` branch.
This is assuming the chart repository will then be https://oracle.github.io/helm-charts

Signed-off: Emmanuel Leroy <emmanuel.leroy@oracle.com>

### CODE REVIEW

**Comment by Djelibeybi** [comment]
We can't merge binaries, they must be published as release artefacts (preferably built by GitHub Actions, but that's not mandatory).

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ products.
 ## Available charts
 
 * Java [Advanced Management Console](./java-amc) (AMC)
+* Oracle SOA Suite [chart](./soa-suite)
```
Comment (Djelibeybi) [inline README.md:9]: Could you please link all of `Oracle SOA Suite` and remove `chart`? It's implied. :)

File: oracledb/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+helm install <deployment_name> . --namespace soans --set 'imagePullSecrets[0].name=image-secret'
```
Comment (Djelibeybi) [inline oracledb/README.md:1]: This isn't enough documentation. We'd need more, including the correct versions of Helm and container images, etc.

File: soa-suite/README.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
+# Deploy Oracle SOA Suite on Kubernetes with the WebLogic Operator
+
+The quick and easy way to deploy this chart on Oracle Kubernetes Engine (OKE) on Oracle Cloud Infrastructure (OCI) is to go check out the related Terraform template here [fixme](fixme), or the step by step guide
```
Comment (Djelibeybi) [inline soa-suite/README.md:3]: Can you please fix the `fixme`?

File: soa-suite/README.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+
+The quick and easy way to deploy this chart on Oracle Kubernetes Engine (OKE) on Oracle Cloud Infrastructure (OCI) is to go check out the related Terraform template here [fixme](fixme), or the step by step guide 
+
+The Terraform template helps deploy some of the Kubernetes dependencies, however if you are deploying on other cloud providers, you will need to provision the persistent storage (as well as the database, for a production deployment) 
+
+To deploy manually, review the instructions below: 
+
+## Disclaimer
+
+This deployment of Oracle SOA Suite makes use of the Oracle SOA Suite Helm Chart based on the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release.
```
Comment (Djelibeybi) [inline soa-suite/README.md:11]: You could combine these two sentences and the Caveats section and just have a single link to the `fmw-kubernetes` distro.

File: soa-suite/README.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+## Caveats
+
+Although this release follows the same flow as the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release, only the Traefik ingress controller is currently supported.
+
+
+## 1. Mandatory Prerequisites
+
+To deploy this chart, you will need:
+
+- An Oracle Container Registry account
```
Comment (Djelibeybi) [inline soa-suite/README.md:24]: This is better referred to as an Oracle Single Sign-on account (which is what OCR users).

File: soa-suite/README.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+- Acknowledge terms of use for the Oracle SOA Suite Docker image on the Oracle Container Registry.
+- Acknowledge terms of use for the Oracle Database Docker image on the Oracle Container Registry to deploy the test Oracle database in your cluster (NOT for production).
+- A Network File Server (NFS) and mount point accessible from within your cluster.
+
+### 1.1 Oracle Container Registry
+
+You will need an Oracle Container Registry account here: [https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::](https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::)
```
Comment (Djelibeybi) [inline soa-suite/README.md:31]: Oracle Container Registry uses Oracle Single Sign-On, so we should probably point folks at https://profile.oracle.com/myprofile/account/create-account.jspx

File: soa-suite/README.md [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+- Click **Enterprise**.
+- Click to accept the License terms and condition on the right.
+- Fill in your information (if you haven't already).
+- Accept the License.
+
+Note that the deployement in cluster is for testing purpose only and not for production.
+
+## 2. Other Prerequisites
+
+To easily deploy the prerequisites through a Terrafrom script, check the related repository here: [fixme](fixme)
```
Comment (Djelibeybi) [inline soa-suite/README.md:63]: Please fix the `fixme`.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]
There are two general requirements:

1. Could you please ensure the copyright header is added to all shell scripts? 
2. Can  you please run `shellcheck` over all the `.sh` files and `markdownlint` over all the MD files?

Thanks!

File: oracledb/README.md (hunk 2 above)
Comment (streamnsight) [reply oracledb/README.md]: reworked the README

File: soa-suite/README.md (hunk 4 above)
Comment (streamnsight) [reply soa-suite/README.md:11]: merged and removed multiple references to fmw-kubernetes

File: soa-suite/README.md (hunk 6 above)
Comment (streamnsight) [reply soa-suite/README.md]: done

File: soa-suite/README.md (hunk 7 above)
Comment (streamnsight) [reply soa-suite/README.md]: fixed the fixme but this is a catch 22: url won't work until the chart is published and i can publish the related terraform.

File: soa-suite/README.md (hunk 5 above)
Comment (streamnsight) [reply soa-suite/README.md]: referenced the SSO portal

**Review by Djelibeybi — APPROVED** [review]

**Pull request merged**